src/main/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStep.java

```diff
@@ -70,19 +68,26 @@ public boolean isChangelog() {
         this.changelog = changelog;
     }
 
+    @Override public StepExecution start(StepContext context) throws Exception {
+        return new StepExecutionImpl(this, context);
+    }
+
     protected abstract @Nonnull SCM createSCM();
 
-    public static final class StepExecutionImpl extends AbstractSynchronousNonBlockingStepExecution<Map<String,String>> {
+    public static final class StepExecutionImpl extends SynchronousNonBlockingStepExecution<Map<String,String>> {
```

Comment:
May be the first step in fixing [JENKINS-44838](https://issues.jenkins-ci.org/browse/JENKINS-44838).